src/Plugin.php

```diff
@@ -42,6 +42,15 @@ public static function preInit() {
     add_filter('woocommerce_variable_price_html', __NAMESPACE__ . '\WooCommerce::woocommerce_variable_sale_price_html', 10, 2);
 
     add_action('parse_request', __CLASS__  . '::parse_request');
+
+    // We need woocommerce-german-market plugin being active to add product
```

Comment:
This can be shortened to a single line:

```suggestion
    // Allow to filter products by delivery time if woocommerce-german-market plugin is active.
```